File: codechef/MSNSADM1.py
# All test cases passed
# https://www.codechef.com/AUG19B/problems/MSNSADM1
# Football

try:
    T = int(input())
    for _ in range(T):
        N = int(input())
        A = [int(x) for x in input().split()]
        B = [int(x) for x in input().split()] 
        C = []

        for i in range(N):
            c = 20*A[i] - 10*B[i]
            C.append(c if c>0 else 0)
        
        print(max(C))
except EOFError as e:
    print(end="")

File: codechef/MSNSADM1.c
// All test cases passed
// https://www.codechef.com/AUG19B/problems/MSNSADM1
// Football

#include<stdio.h>

int main(){
    int T, N;
    scanf("%d",&T);
    while(T--){
        scanf("%d",&N);
        int A[N], B[N], C[N];
        for(int i=0;i<N;i++)
            scanf("%d",&A[i]);
        for(int i=0;i<N;i++)
            scanf("%d",&B[i]);
        for(int i=0;i<N;i++){
            int c = 20*A[i] - 10*B[i];
            C[i] = (c>=0 ? c : 0);
        }

        int max = C[0];
        for(int i=1;i<N;i++){
            if(C[i] > max)
                max = C[i];
        }
            
        printf("%d\n",max);
            
    }
    return 0;
}